components/Tabs/Tabs.js

```diff
@@ -1,51 +1,63 @@
-
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element = element;
+
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = this.element.dataset.tab;
+
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(
+      `.tab-item[data-tab="${this.data}"]`
+    );
+
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
+    this.tabItem = new TabItem(this.itemElement);
+    //Why itemElement???
```

Comment:
because itemElement refers to the specific data-tab of the tabItem we are looking for. so when we click on the link, it will refer to the correct content.